src/http/server/handlers/static_file_handler.cr

```diff
@@ -71,7 +72,7 @@ class HTTP::StaticFileHandler
       context.response.content_type = "text/html"
       directory_listing(context.response, request_path, file_path)
     elsif is_file
-      last_modified = modification_time(file_path)
+      last_modified = file_info.not_nil!.modification_time
```

Comment:
I think the is_file check can be inlined here, so there's no need for not_nil!

Reply:
Oh yeah, that's a good alternative to early return.

Reply:
Compiler nil check error without an early return